docs/integrations/vector-db-integrations/couchbase.mdx

```diff
@@ -0,0 +1,133 @@
+---
+title: Couchbase
+sidebarTitle: Couchbase
+---
+
+
+
+This is the implementation of the Couchbase Vector store data handler for MindsDB.
+
+[Couchbase](https://www.couchbase.com/) is an open-source, distributed multi-model NoSQL document-oriented database software package optimized for interactive applications. These applications may serve many concurrent users by creating, storing, retrieving, aggregating, manipulating, and presenting data.
+
+## Prerequisites
+
+Before proceeding, ensure the following prerequisites are met:
+
+1. Install MindsDB locally via [Docker](/setup/self-hosted/docker) or [Docker Desktop](/setup/self-hosted/docker-desktop).
+2. To connect Couchbase to MindsDB, install the required dependencies following [this instruction](/setup/self-hosted/docker#install-dependencies).
+3. Install or ensure access to Couchbase.
+
+## Implementation
+
+This handler is implemented using the `couchbase` library, the Python driver for Couchbase.
+
+The required arguments to establish a connection are as follows:
+* `connection_string`: the connection string for the endpoint of the Couchbase server
+* `bucket`: the bucket name to use when connecting with the Couchbase server
+* `user`: the user to authenticate with the Couchbase server
+* `password`: the password to authenticate the user with the Couchbase server
+* `scope`:  scopes are a level of data organization within a bucket. If omitted, will default to `_default`
+
+Note: The connection string expects either the couchbases:// or couchbase:// protocol.
+
+<Tip>
+If you are using Couchbase Capella, you can find the connection_string under the Connect tab.
+It will also be required to whitelist the machine(s) that will be running MindsDB and database credentials will need to be created for the user. These steps can also be taken under the Connect tab.
+</Tip>
+
+In order to make use of this handler and connect to a Couchbase server in MindsDB, the following syntax can be used. Note, that the example uses the default `travel-sample` bucket which can be enabled from the couchbase UI with pre-defined scope and documents. 
+
+```sql
+CREATE DATABASE couchbase_vectorsource
+WITH
+engine='couchbase',
+parameters={
+    "connection_string": "couchbase://localhost",
+    "bucket": "travel-sample",
+    "user": "admin",
+    "password": "password",
+    "scope": "inventory"
+};
+```
+
+## Usage
+
+Now, you can use the established connection to create a collection (or table in the context of MindsDB) in Couchbase and insert data into it:
+
+### Creating connection
```

Comment:
I think we can remove this section `### Creating connection` because it is shown just above how to connect, so let's continue with examples here.